### PR TITLE
Run prettier 0.20.0

### PR DESCRIPTION
**Pull request by vjeux**

### CODE REVIEW

File: src/printer.js [hunk 1]
```diff
@@ -1920,10 +1925,10 @@ function printArgumentsList(path, options, print) {
           lastArg.body.type === "ObjectExpression" ||
           lastArg.body.type === "ArrayExpression" ||
           lastArg.body.type === "CallExpression" ||
-         lastArg.body.type === "JSXElement"))) &&
-        // If the last two arguments are of the same type,
-        // disable last element expansion.
-        (!penultimateArg || penultimateArg.type !== lastArg.type);
```
Comment (vjeux) [inline src/printer.js]: It's so nice not to have to care about those kind of things during review and know that it'll be correctly formatted eventually :)

Comment (umidbekkarimov) [inline src/printer.js]: yep, `0.20.0` is a killer release, now my code is just perfect

Comment (jlongster) [inline src/printer.js]: 👍 Awesome!

**Comment by jlongster** [comment]
Can you do an `amend` to create a new commit and force push to trigger the CI? I can't seem to re-trigger it from their interface; it's like that push is corrupt or something from the aws outage.

**Pull request merged**